index.html

```diff
@@ -0,0 +1,82 @@
+<!doctype html>
+
+<html lang="en">
+
+ <head>
+  <meta charset="UTF-8">
+  <meta name="Generator" content="EditPlus®">
+  <meta name="Author" content="Twilight">
+  <title>Todolist</title>
+ </head>
+
+ <body>
+  <h3>·Todolist</h3>
+  <h4>·Append:</h4>
+  <form action="">
+	&nbsp;&nbsp;&nbsp;&nbsp;<input id="add_todolist"></input>
```

Comment:
Format: Use **consistent** style for indenting

i.e. don't mix spaces with tabs; and use the same amount of spaces for each indentatation